src/textual/containers.py

```diff
@@ -29,6 +29,10 @@ class Container(Widget):
 class ScrollableContainer(Widget, can_focus=True, inherit_bindings=False):
     """A scrollable container with vertical layout, and auto scrollbars on both axis."""
 
+    # We don't typically want to maximize scrollable containers,
+    # since the user can easily navigate the contents
+    ALLOW_MAXIMIZE = False
```

Comment:
Not sure about this comment. The areas of Posting I've had people ask most for "maximise" in the past are scrollable. They just want to see more on screen at once.

Reply:
But most containers we wouldn't want to be maximized. Apps tend to have a lot of containers, and we wouldn't want them to maximized for the most part. But you can always set that to `True` when you do.

Reply:
Isn't your comment here at odds with the current implementation though? Containers currently can be maximised if they're focusable?